rm/rm-node/src/main/java/org/ow2/proactive/resourcemanager/utils/RRDSigarDataStore.java

```diff
@@ -182,23 +186,23 @@ void sample(RrdDb dataBase, long timeInMs) throws IOException {
                 }
             } catch (NumberFormatException ex) {
                 // do not save non-numeric values
-                if (logger.isTraceEnabled()) {
+                String exceptionAsString = StackTraceUtil.getStackTrace(ex).replaceAll("For input string: \".*\"", "");
+                exceptionAsString = dataSource + " / " + fullName + exceptionAsString;
```

Comment:
I suggest to simply use: exceptionAsString = fullName + " / " + exceptionAsString;